packages/crud/test/a11y.test.js

```diff
@@ -256,4 +256,39 @@ describe('a11y', () => {
       </vaadin-crud>
   `),
   );
+
+  ['aside', 'bottom'].forEach((editorPosition) => {
+    describe(`editor focus - ${editorPosition} position`, () => {
+      let newButton, editButtons, editor;
+
+      beforeEach(async () => {
+        crud = fixtureSync('<vaadin-crud></vaadin-crud>');
+        crud.editorPosition = editorPosition;
+        crud.items = [{ title: 'Item 1' }];
+        await nextRender();
+        newButton = crud.querySelector('[slot=new-button]');
+        editButtons = crud.querySelectorAll('vaadin-crud-edit');
```

Comment:
```suggestion
        editButtons = [...crud.querySelectorAll('vaadin-crud-edit')];
```

Reply:
This isn't needed as we don't use array methods, same as in focus restoration suites above.